test/fixtures/poro.rb

```diff
@@ -70,10 +70,21 @@ def cache_key
 
 class Employee < ActiveRecord::Base
   has_many :pictures, as: :imageable
+  has_many :object_tags, as: :taggable
+end
+
+class ObjectTag < ActiveRecord::Base
+  belongs_to :poly_tag
+  belongs_to :taggable, polymorphic: true
 end
 
 class Picture < ActiveRecord::Base
   belongs_to :imageable, polymorphic: true
+  has_many :object_tags, as: :taggable
+end
+
+class PolyTag < ActiveRecord::Base
+  has_many :object_tags
```

Comment:
There were other tests using a non-active record version of Tags.  Didnt want to do to much upheaval to that pattern so made a new Tag Class